src/job.ts

```diff
@@ -255,7 +259,7 @@ export class CronJob<OC extends CronOnCompleteCommand | null = null, C = null> {
 				this.running = false;
 
 				// start before calling back so the callbacks have the ability to stop the cron job
-				if (!this.cronTime.realDate) {
+				if (!this.runOnce) {
```

Comment:
should this be `this.runOnce()`?

Reply:
no, this is the right way, getters are accessed as normal properties and are not callable